docs/source/object_detection.mdx

```diff
@@ -154,6 +154,9 @@ You can verify the transform works by visualizing the 10th example:
 ... )
 ```
 
+To learn how to leverage parts of 🤗 datasets end-to-end for training an object detection model, refer to
+[this notebook](https://nbviewer.org/github/NielsRogge/Transformers-Tutorials/blob/master/YOLOS/Fine_tuning_YOLOS_for_object_detection_on_custom_dataset_%28balloon%29.ipynb).
```

Comment:
Any reason why you opted for nbviewer for this link, but GitHub for the other?

Reply:
The notebook has a couple of visualisations which is probably why it is not rendering on GitHub fully. Hence.

Reply:
sg, that's fine